src/cli/cli.cpp

```diff
@@ -554,6 +595,7 @@ void Interpreter::ProcessCounters(int argc, char *argv[])
             sServer->OutputFormat("    RxErrSec: %d\r\n", counters->mRxErrSec);
             sServer->OutputFormat("    RxErrFcs: %d\r\n", counters->mRxErrFcs);
             sServer->OutputFormat("    RxErrOther: %d\r\n", counters->mRxErrOther);
+            otFreeMemory(counters);
```

Comment:
Yes, I could define an `otPtr` class that we use in the CLI. On Windows it would be a smart pointer that calls `otFreeMemory` in the destructor. Everywhere else, it would do nothing. Would that work for you?